assets/src/js/product-editor.js

```
@@ -957,14 +959,14 @@
                 type: 'GET',
                 data: {
                     product_id: product_id,
-                    amount: sale_price ? sale_price : product_price,
+                    amount: priceAmount ? Dokan_Editor.unformatMoney( priceAmount ) : 0,
                     // vendor_id
                     category_ids,
                     context: 'seller'
                 }
             }).done( ( response ) => {
                 if ( ! isNaN( response ) ) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)